config/settings_data.json

```diff
@@ -390,7 +390,8 @@
   "platform_customizations": {
     "custom_css": [
       ".icon-cart .cart_count {display: none;}",
-      ".nav a.icon-cart span {display: none;}"
+      ".nav a.icon-cart span {display: none;}",
+      "ul.exhibition-list {display:none;}"
```

Comment:
We can remove the exhibition list from the code view unless it's helpful for debug. No harm in keeping and hiding like this.